pkg/controller/installation/core_controller.go

```diff
@@ -538,7 +538,8 @@ func mergeProvider(cr *operator.Installation, provider operator.Provider) error
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
 func (r *ReconcileInstallation) Reconcile(request reconcile.Request) (reconcile.Result, error) {
 	reqLogger := log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
-	reqLogger.V(1).Info("Reconciling Installation.operator.tigera.io")
+	reqLogger.Info("Reconciling Installation.operator.tigera.io")
+	defer reqLogger.Info("Finished reconciling network installation")
```

Comment:
Is there a way to tell now when we have reached the end of this function vs. exiting early for some reason?

Reply:
Not currently.
Should I add a flag that we set at the end before returning to be able to tell that difference?
Or just drop this change. The thing is I don't think we always have a message when the Reconcile finishes (early).

Reply:
I don't think it's really worth adding a flag.

Either leave it as it is in this PR if you think it's worthwhile, or revert and add logging to the places where it is missing would be my suggestion.